src/olympia/devhub/templates/devhub/addons/includes/static_theme_preview_svg.xml

```diff
@@ -1,5 +1,5 @@
 {% autoescape true %}
-<svg id="Svg1006" width="680" height="100" xmlns="http://www.w3.org/2000/svg"
+<svg id="svg" width="680" height="100" xmlns="http://www.w3.org/2000/svg"
```

Comment:
nit: could use a less generic id :)

Reply:
I almost called it "svg-svg" :)

Reply:
https://github.com/mozilla/addons-server/pull/7447/commits/f42db288c018dec51d72c9adf3662cc3226b5ffd